### ISSUE TITLE
blob deletion doesn't work

### ISSUE THREAD

**Issue by tlievre** [Azure-Samples/azure-search-openai-demo#1806]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> launch prepdocs.ps1 script with `--remove` argument and path to the file

### Any log messages given by the failure
> Using local files: C:\xxx\azure-search-openai-demo/data/xxx/xxx xxx_v70.08.228.00.pdf
Removing sections from '{C:\xxx\azure-search-openai-demo/data/xxx/xxx xxx_v70.08.228.00.pdf or '<all>'}' from search index 'gptkbindex'
Removed 20 sections from index

### Expected/desired behavior
> pdf should be delete from blob storage

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.

### Versions
> 1.9.4 (commit 60d7a770c73289e303a539babf5965e638843227)

### Mention any other details that might be useful

![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/62014278/4451501f-359d-4ac6-99a9-305021cd685b)

This condition in remove_blob method from blobManager class is always raised so blob deletion never occur.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by mattgotteiner** [comment]
Yeah, there's definitely a problem with both the shell script and the prepdocs code using this specific file pattern name
